code/modules/projectiles/gun_attachables.dm

```diff
@@ -499,7 +499,7 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 	flags_attach_features = NONE
 
 /obj/item/attachable/t29barrel
```

Comment:
paths should be updated to match their correct designation now (sg29barrel instead of t29 and so on)

Reply:
I'll do that if I have extra time